edit-post/store/defaults.js

```diff
@@ -1,9 +1,10 @@
 export const PREFERENCES_DEFAULTS = {
-	mode: 'visual',
-	sidebars: {
-		desktop: true,
-		mobile: false,
-		publish: false,
+	editorMode: 'visual',
+	viewportType: 'desktop', // 'desktop' | 'mobile'
+	activeGeneralSidebar: null, // null | 'editor' | 'plugins'
```

Comment:
Looks like here we should set activeGeneralSidebar to 'editor', otherwise we are introducing a behavior change because when reloading the page no sidebar will be active.